evxtb/xtb_ev.py

```diff
@@ -1,12 +1,33 @@
+import os
 import re
 import subprocess
+from contextlib import contextmanager
+from pathlib import Path
+from typing import Generator
 
 from ase.eos import EquationOfState
 from ase.units import kJ, Hartree, eV
 import matplotlib.pyplot as plt
 import numpy as np
 
 
+@contextmanager
+def tempfile(fname: str = "temp") -> Generator[Path, None, None]:
+    """Create a temporary file that is deleted on return.
+
+    Args:
+        fname: The name of the file to create.
+
+    """
+    fpath = Path(fname)
+    with fpath.open("w"):
+        pass
+    try:
+        yield Path(fname)
+    finally:
+        os.remove(fpath)
+
+
```

Comment:
This is the core of the new functionality: the `contextmanager` that handles creation and deletion of the file.